### PR TITLE
Unbundle parameters into client files

### PR DESCRIPTION
**Pull request by patilsnr**

### CODE REVIEW

File: iothub/service/src/Jobs/ScheduledJobsClient.cs [hunk 1]
```diff
@@ -192,18 +192,19 @@ public virtual async Task<ScheduledJob> CancelAsync(string jobId, CancellationTo
         /// </exception>
         /// <exception cref="OperationCanceledException">If the provided <paramref name="cancellationToken"/> has requested cancellation.</exception>
         public virtual async Task<ScheduledJob> ScheduleDirectMethodAsync(
-            ScheduledDirectMethod scheduledDirectMethod,
+            //ScheduledDirectMethod scheduledDirectMethod,
+            string queryCondition,
+            DirectMethodRequest directMethodRequest,
+            DateTime startOnUtc,
```
Comment (drwill-ms) [inline iothub/service/src/Jobs/ScheduledJobsClient.cs:198]: DateTimeOffset?

**Review by drwill-ms — APPROVED** [review]

**Review by tmahmood-microsoft — APPROVED** [review]

**Pull request merged**